paxml/summary_utils.py

```diff
@@ -396,6 +398,12 @@ def write_summary_entry(summary_writer: SummaryWriter,
       status_msg += f', steps/sec {steps_per_sec}'
       write_summary_tensor(step_i, 'Steps/sec', steps_per_sec,
                            SummaryType.AGGREGATE_SCALAR)
+
+      if global_bs is not None:
+        seqs_per_sec = num_sub_batches*global_bs*steps_per_sec
```

Comment:
why is this seqs_per_sec, we are not multiplying by seq_len

Reply:
nvm, we don't want to multiply by seq_len, otherwise it'd be tokens/sec. This is resolved.